### DISCUSSION TITLE
trigger webhook only on release/tag

### DISCUSSION THREAD

**Discussion by narasimhamurthi** [community/community#51682] [Apps, API and Webhooks]
### Select Topic Area

General

### Body

i would like to trigger the webhook on release/tags. 

i am getting the error as attached.
![github-error](https://user-images.githubusercontent.com/111184660/229293965-6a813729-9e55-4b21-aea3-2015619a5e94.PNG)

**Comment by airtower-luna** [comment]
As it's in the response, that error seems to be from the service you're sending the webhook *to*. And the message seems pretty clear that that service (whatever it is) can't handle that event.

**Comment by narasimhamurthi** [comment]
okay.
